infrastructure/src/main/java/org/wso2/testgrid/infrastructure/CloudFormationScriptPreprocessor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.testgrid.infrastructure;
+
+import org.wso2.testgrid.common.util.StringUtil;
+
```

Comment:
Can avoid the new line?

Reply:
Hi Lahiru, This new line is due to IDE configuration where it adds a new line between  (java.___ , javax.__ imports ) and (other imports). 
AFAIK WSO2 uses the configuration. :)

![te](https://user-images.githubusercontent.com/8229975/36408220-28a5a33c-162a-11e8-84fe-e98b0ea59f25.png)